docs/source/releases/v1_11_3a0.rst

```diff
@@ -60,6 +61,17 @@ has been resolved.
 
   modified: pyproject.toml
 
+Add PluginError for empty YAML plugins
+--------------------------------------
+
+Currently, when a YAML plugin is empty in GeoIPS, the error raised is non-descriptive
+and not helpful. To fix this, we've added a PluginError which points the user to the
+empty YAML file, and tells them to fill it with the appropriate information.
+
+::
+
+    modified: geoips/geoips/geoips_utis.py
```

Comment:
Do we want to add a test for this?

Reply:
What kind of test are you thinking here?

Reply:
An empty YAML file, but I do not really care.  Just want to make sure we are on the same page :-)

Reply:
I think it's good for now.